brdnet/classifier.py

```diff
@@ -128,34 +135,50 @@ def load_data(args):
     val_Y: numpy.array
         The labels for val_X
     '''
-    Z_df = pandas.read_csv(args.Z_file_path, sep='\t')
+    Z_df = pandas.read_csv(Z_file_path, sep='\t')
 
     # Ensure the gene symbols are in alphabetical order
     Z_df = Z_df.sort_index()
 
-    healthy_df = pandas.read_csv(args.healthy_file_path, sep='\t')
-    disease_df = pandas.read_csv(args.disease_file_path, sep='\t')
+    healthy_df = pandas.read_csv(healthy_file_path, sep='\t')
+    disease_df = pandas.read_csv(disease_file_path, sep='\t')
 
-    return prepare_input_data(Z_df, healthy_df, disease_df, args.seed)
+    return prepare_input_data(Z_df, healthy_df, disease_df, seed)
 
 
-def train_model(train_X, train_Y, val_X, val_Y,
-                model_name=None, logdir=None, lr=None, epochs=None, batch_size=16):
-    # Create log directory
-    os.mkdir(logdir)
-    model_name = model_name
+def train_model(train_X, train_Y, val_X, val_Y, checkpoint_path,
```

Comment:
Just curious, why did you choose those 3 models to start with?

Reply:
I wanted to see how performance would change with progressively more complicated models, so I started with Logistic Regression, scaled up to a small multilayer perceptron, then tried a similar ten-layer network